### ISSUE TITLE
passenger-status cannot find suitable Ruby interpreter

### ISSUE THREAD

**Issue by carlosbustillordguez** [phusion/passenger-docker#313]
When I run `passenger-status` I get the following:

```bash
passenger_system_ruby: cannot find suitable Ruby interpreter
````

It seems `passenger-status` uses `/usr/bin/passenger_system_ruby` which is not working.

I am using the Docker image `phusion/passenger-ruby30:1.0.14`.

As a workaround I replaced the original `/usr/bin/passenger_system_ruby` with the system ruby version:

```bash
ln -sf /usr/bin/ruby3.0 /usr/bin/passenger_system_ruby
```

Also, I tested on others images like `phusion/passenger-ruby27:1.0.14` and `phusion/passenger-ruby25:1.0.14`, in both images the `passenger-status` command works as expected.

**Comment by CamJN** [comment]
Ah I found the issue in `passenger_system_ruby`, it wasn't adjusted for the 2.7 -> 3.0 jump, it expected 2.8 next. I'll see what I can do for 1.0.15 since passenger 6.0.9 just came out without a fix.

**Comment by CamJN** [comment]
Ok I've included a workaround in 1.0.15.

**Comment by carlosbustillordguez** [comment]
> Ok I've included a workaround in 1.0.15.

Thanks!!
